dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/importers/ui/form/FormConstants.java

```diff
@@ -0,0 +1,11 @@
+package edu.harvard.iq.dataverse.importers.ui.form;
+
+public class FormConstants {
```

Comment:
Similar opinion to the one before, enums are better imo.

Reply:
But unfortunately again there are non-homogenic types of constants here.